src/gui/wizard/screen_wizard.cpp

```diff
@@ -181,7 +181,7 @@ string_view_utf8 WizardGetCaption(WizardState_t st) {
     }
 
     if (IsStateInWizardMask(st, WizardMaskFirstLay())) {
-        return _("FIRST LAYER CALIB.");
+        return _("FIRST LAYER CAL.");
```

Comment:
please document above this line of code, how many letters is this string allowed to have (for translations)